hack/containerfile-linter.sh

```
@@ -2,7 +2,7 @@
 
 declare -a CONTAINERFILES
 
-CERT_MANAGER_OPERATOR_CONTAINERFILES=("Containerfile.cert-manager" "Containerfile.cert-manager.acmesolver" "Containerfile.cert-manager-operator" "Containerfile.cert-manager-operator.bundle")
+CERT_MANAGER_OPERATOR_CONTAINERFILES=("Containerfile.cert-manager" "Containerfile.cert-manager.acmesolver" "Containerfile.cert-manager-operator" "Containerfile.cert-manager-operator.bundle" "Containerfile.istio-csr")
```

Not sure why each file was listed earlier. Please update it to

```suggestion
CERT_MANAGER_OPERATOR_CONTAINERFILES=($(ls Containerfile.*))
```